Domains/Grids/Grids.ecschema.xml

```diff
@@ -3,59 +3,15 @@
         <BaseClass>bis:ISubModeledElement</BaseClass>
     </ECEntityClass>
 
-    <ECEntityClass typeName="GridCurveBundle" displayLabel="GridCurve bundle" description="a driver bundle element between surfaces and the GridCurve elements">
-        <BaseClass>bis:DriverBundleElement</BaseClass>
-        <ECNavigationProperty propertyName="CurvesSet" relationshipName="GridCurveBundleRefersToGridCurvesSet" direction="Forward" description="Created GridCurve elements will be put into this GridCurvesSet" />
-    </ECEntityClass>
-
-    <ECRelationshipClass typeName="GridCurveBundleRefersToGridCurvesSet" strength="embedding" strengthDirection="Backward" modifier="None" description="Maps GridCurveBundle to a GridCurvesSet">
-        <Source multiplicity="(0..*)" roleLabel="puts GridCurve elements into" polymorphic="true">
-            <Class class="GridCurveBundle"/>
-        </Source>
-        <Target multiplicity="(1..1)" roleLabel="contains GridCurve elements created by" polymorphic="true">
-            <Class class="GridCurvesSet"/>
-        </Target>
-    </ECRelationshipClass>
-
-    <ECRelationshipClass typeName="GridSurfaceDrivesGridCurveBundle" modifier="None" strength="referencing" description="a driving relationship which tells that gridsurface influences the creation of GridCurve">
-        <BaseClass>bis:ElementDrivesElement</BaseClass>
-        <Source multiplicity="(0..*)" roleLabel="influences" polymorphic="true">
-            <Class class="GridSurface"/>
-        </Source>
-        <Target multiplicity="(0..*)" roleLabel="is influenced by" polymorphic="true">
-            <Class class="GridCurveBundle"/>
-        </Target>
-    </ECRelationshipClass>
-
-    <ECRelationshipClass typeName="GridCurveBundleCreatesGridCurve" modifier="None" strength="referencing" description="a driving relationship which tells that GridCurve was created by the mapped GridCurveBundle">
-        <BaseClass>bis:ElementDrivesElement</BaseClass>
-        <Source multiplicity="(1..1)" roleLabel="creates" polymorphic="true">
-            <Class class="GridCurveBundle"/>
-        </Source>
-        <Target multiplicity="(0..1)" roleLabel="is created by" polymorphic="true">
-            <Class class="GridCurve"/>
-        </Target>
-    </ECRelationshipClass>
-
-    <ECRelationshipClass typeName="GridDrivesGridSurface" modifier="None" strength="referencing" description="a driving relationship which tells that a grid is driving a gridsurface.">
-        <BaseClass>bis:ElementDrivesElement</BaseClass>
-        <Source multiplicity="(1..1)" roleLabel="drives" polymorphic="true">
-            <Class class="Grid"/>
-        </Source>
-        <Target multiplicity="(0..*)" roleLabel="is driven by" polymorphic="true">
-            <Class class="GridSurface"/>
-        </Target>
-    </ECRelationshipClass>
-
     <ECRelationshipClass typeName="GridAxisContainsGridSurfaces" modifier="None" strength="embedding" description="maps axis to grouped surfaces">

```

Comment:
I believe your question on email regarding whether an InformationRecord could own a SpatialLocation element corresponds to this relationship, correct? In BIS terms, if this case is truly a "parent owns children" situation, the parent (GridAxis) will be forced to be a SpatialLocationElement. 

The current "contained in" role seems strange too, considering it is defined between an InformationRecord and a SpatialLocation element.

I haven't wrapped my head around the concept of a GridAxis and why it is introduced as an InformationRecord. I'll need to catch up on that design decision in order to have a stronger opinion about what BIS relationship is more appropriate in this case.